src/lib/emojiKeywords.js

```diff
@@ -236,5 +236,7 @@ export default {
   "orpheus podcast": "tos-icon",
   podcast: "studio_microphone",
   quest: "quests",
-  puzzmo: "puzzmo"
+  puzzmo: "puzzmo",
+  "purple bubble": "purplebubble",
+  purplebubble:purplebubble
```

Comment:
i think you should put the value of 'purplebubble' within quotes

Reply:
got it! thanks!